.github/workflows/ci.yml

```diff
@@ -11,41 +11,41 @@ jobs:
-        E2E_TEST_ESS_VC_SERVICE: https://consent.pod.inrupt.com
-        E2E_TEST_ESS_VC_SUBJECT: https://pod.inrupt.com/solid-client-e2e-tester-ess/profile/card#me
-        E2E_TEST_ESS_CLIENT_ID: ${{ secrets.POD_SPACES_CLIENT_ID }}
-        E2E_TEST_ESS_CLIENT_SECRET: ${{ secrets.POD_SPACES_CLIENT_SECRET }}
-        E2E_TEST_DEV_NEXT_IDP_URL: ${{ secrets.E2E_TEST_DEV_NEXT_IDP_URL }}
-        E2E_TEST_DEV_NEXT_VC_SERVICE: https://vc.dev-next.inrupt.com
-        E2E_TEST_DEV_NEXT_VC_SUBJECT: https://id.dev-next.inrupt.com/sdkdevnexte2e
-        E2E_TEST_DEV_NEXT_CLIENT_ID: ${{ secrets.E2E_TEST_DEV_NEXT_CLIENT_ID }}
-        E2E_TEST_DEV_NEXT_CLIENT_SECRET: ${{ secrets.E2E_TEST_DEV_NEXT_CLIENT_SECRET }}
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: "npm"
+          cache-dependency-path: "**/package-lock.json"
+      - run: npm ci
+      - run: npm run build
+      - run: npm run test
+      # TODO: Replace with shared auditing:
+      - run: npm run check-licenses
+      - run: npm run e2e-test-node
+        # To prevent conflicts of multiple jobs trying to modify the same Resource at the same time,
+        # and because behaviour on different OS's is already tested by unit tests,
+        # end-to-end tests only need to run on one OS:
+        if: runner.os == 'Linux' && matrix.node-version == '16.x' && github.actor != 'dependabot[bot]'
+        env:
+          E2E_TEST_ESS_IDP_URL: https://broker.pod.inrupt.com
+          E2E_TEST_ESS_VC_SERVICE: https://consent.pod.inrupt.com
+          E2E_TEST_ESS_VC_SUBJECT: https://pod.inrupt.com/solid-client-e2e-tester-ess/profile/card#me
+          E2E_TEST_ESS_CLIENT_ID: ${{ secrets.POD_SPACES_CLIENT_ID }}
+          E2E_TEST_ESS_CLIENT_SECRET: ${{ secrets.POD_SPACES_CLIENT_SECRET }}
+          E2E_TEST_DEV_NEXT_IDP_URL: ${{ secrets.E2E_TEST_DEV_NEXT_IDP_URL }}
+          E2E_TEST_DEV_NEXT_VC_SERVICE: https://vc.dev-next.inrupt.com
+          E2E_TEST_DEV_NEXT_VC_SUBJECT: https://id.dev-next.inrupt.com/sdkdevnexte2e
+          E2E_TEST_DEV_NEXT_CLIENT_ID: ${{ secrets.E2E_TEST_DEV_NEXT_CLIENT_ID }}
+          E2E_TEST_DEV_NEXT_CLIENT_SECRET: ${{ secrets.E2E_TEST_DEV_NEXT_CLIENT_SECRET }}
     # Ideally, we'd autodetect installed browsers and run them headlessly.
     # See https://github.com/DevExpress/testcafe/issues/5641
-    - name: Prepare browser-based end-to-end tests
-      run: |
-        cd .codesandbox/sandbox
-        npm install
-        # Run the end-to-end tests against the code in this branch specifically:
-        npm install ../../
-        cd ../..
+    # - name: Prepare browser-based end-to-end tests
+    #   run: |
+    #     cd .codesandbox/sandbox
+    #     npm install
+    #     # Run the end-to-end tests against the code in this branch specifically:
+    #     npm install ../../
+    #     cd ../..
```

Comment:
Is there a reason for not just deleting this since the plan is to remove all codesandbox stuff anyways?

Reply:
Yeah, it's purely to have the reminder to go back and do work here.